### DISCUSSION TITLE
Studio Quantum: Quantum Technology in Art and Science, Talk Series at ZKM | Hertzlab [Karlsruhe, DE] (4.9.-30.10.2024)

### DISCUSSION THREAD

**Discussion by studiofomalhaut** [Quantumland-art/community#79] [Art]
**"Since its beginnings, quantum physics has revolutionized the world of ideas in science. Recently, technologies such as the quantum computer have promised new spurts of innovation. But for most people, the science of the quantum remains one thing above all - mysterious. A residency program and a series of events at the ZKM | Hertzlab aim to shed new light on the subject.**

The new residency program “Studio Quantum” explores how art can help to better understand the abstract topic of quantum research. The aim is to create connections between artists and partners from culture and science, the education sector and the tech industry. The aim is to promote a sustainable dialogue on the subject of quantum technology and art through public exchange.

Around the program, a series of events with guests from art, science and research will provide insights into the fascinating field of quantum physics and quantum technologies. To kick things off, we will be talking to Prof. Dr. Philip Willke, Tenure Track Professor at the Karlsruhe Institute of Technology (KIT), and Adrian Schmidt from the KIT / ITAS Institute for Technology Assessment and Systems Analysis.

The residency program and the event series take place in cooperation with the Goethe-Institut and the KIT / ITAS Institute for Technology Assessment and Systems Analysis."

Find out more [here](https://zkm.de/en/2024/09/studio-quantum-quantum-technology-in-art-and-science).

**Comment by studiofomalhaut** [comment]
This may sound a bit repetitive in relation to previous posts. Studio Quantum is currently the most active program developing the intersection beetween quantum technology and art, and the residencies take place in different places around the world. Great to see that ZKM has a program dedicated to the topic though!

The last talk is today! So I'm a little late on spotting this talk series.

**Reply by omarcostahamido** [reply]
@studiofomalhaut 
Absolutely no worries. If they are very active then it is to be expected for that to reflect here. The most important thing is to know what is happening and get a better sense of how this field is progressing :) Though I wish I could be there and listen to the talks. I am very curious to know what exactly are the `new techniques and methods of artistic production enabled by quantum technologies.` they keep mentioning. To quote the structure of the [community](https://community.quantumland.art/database) database, a full picture of `who is`, `where was the event`, `what was the creative work`, `were there any writings`, and `which new software was used` 😉

**Reply by studiofomalhaut** [reply]
@omarcostahamido ok, I'll take note of the structure! Indeed, the descriptions are broad and do not mention specific technologies.

**Reply by omarcostahamido** [reply]
yeah, often it is. that's why we need more ppl interacting here. hopefully some will be more directly involved with these events and can translate that knowledge into a pull request on the database 🤞

**Reply by Itaborala** [reply]
I'm also very curious about the new methods! I could assist someone that was directly involved in including this event and related work into the database, since I had some experience in updating this database in the past.

**Reply by omarcostahamido** [reply]
thank you @Itaborala 🙏 
and you make a great point... there's definitely some `getting started` documentation lacking on the [FAQs](https://community.quantumland.art/faqs) - _tapping myself on the back_.
I also thought at some point to include an easier way for non-tech savy folks to contribute things, like an online form submission page. And as a matter of fact, the older version of this, in notion, used a special built-in form to collect contributions. This was before notion had forms, which they [just announced](https://www.youtube.com/watch?v=FEy6f8U212s), it was called chilly something 😄, and its submissions would directly populate a notion database. Huh... as a matter of fact there's still some hidden remains of it on [this page](https://community.quantumland.art/faq/contact-form), but it should be retired sometime in the near future. But, and especially for this new version of the database, I see a couple problems with relying on an external online form submission page: 
- who hosts and manages the form submissions? like which google account for google forms, or airtable subscription, etc.
- even though you could share access to it, it doesn't change the fact that it complicates the stack of tools being used to manage this, and distributes information on different platforms.
- how can you create a transparent and responsible record of what is changed in the database? this one is more directed at _reasons why I had to drop notion_: there you just add ppl to the workspace and then they can make whatever changes they want afaik 😕
- and let's not forget the elephant in the room: notion is not open source, and you can't really easily protect your data. Sure their AI and search capabilities may be good, but there might be some helpful convincing alternative on that too #32. community.quantumland.art should really strive to stick with its opensource/openaccess statement (on the [homepage](https://community.quantumland.art/) haha). 

TLDR: community.quantumland.art is an ever evolving project in open source and open access. Contributing to the database directly requires a little bit of tech savyness at the moment, but we are very blessed to have a small community of people willing to help (not unlike other open source projects). Contributing and interacting on the discussions forum only requires a free github account, and this would be the best place to start and also ask for help on how to directly contribute to the database.

(this refers to #10 )
